Corzbank.Services/GenericService.cs

```diff
@@ -36,6 +36,11 @@ public async Task<T> Get(int id)
 			return await _dbContext.Set<T>().FindAsync(id);
 		}
 
+		public async Task<T> GetByGuid(Guid id)
+		{
+			return await _dbContext.Set<T>().FindAsync(id.ToString());
+		}
```

Comment:
What's the difference between Get and that method? Generic service should not contain a million methods, here you should declare only necessary ones.

Reply:
resolved